packages/appium/lib/cli/npm.js

```diff
@@ -169,13 +187,13 @@ export default class NPM {
     // everything got installed ok. Remember, pkgName might end up with a / in it due to an npm
     // org, so if so, that will get correctly exploded into multiple directories, by path.resolve here
     // (even on Windows!)
-    const pkgJson = path.resolve(pkgDir, 'node_modules', pkgName, 'package.json');
+    const pkgJsonPath = resolveFrom(this.appiumHome, `${pkgName}/package.json`);
```

Comment:
based on the comment above is it ok to explicitly use slash as a path separator? would that still work on Windows?

Reply:
same below

Reply:
so, yes, because `resolveFrom` uses `require.resolve()` under the hood (which, in turn, is used by `require()`) which normalizes paths.  

If I wanted to build a path but _not_ attempt to resolve it as a module, I'd need to use e.g., `path.join()` or `path.resolve()` and pass multiple args without slashes.